bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Znunugamma_aTGC/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets_run_card.dat

```diff
@@ -0,0 +1,253 @@
+  91.1880	=  scale             ! fixed ren scale
+  91.1880	=  dsqrt_q2fact1     ! fixed fact scale for pdf1
+  91.1880	=  dsqrt_q2fact2     ! fixed fact scale for pdf2
+  1	=  scalefact         ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  1	=  ickkw             ! 0 no matching, 1 MLM, 2 CKKW matching
+  1	=  highestmult       ! for ickkw=2, highest mult group
+  1	=  ktscheme          ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1	=  alpsfact          ! scale factor for QCD emission vx
+  F	=  chcluster         ! cluster only according to channel diag
+  F	=  pdfwgt            ! for ickkw=1, perform pdf reweighting
+  5	=  asrwgtflavor      ! highest quark flavor for a_s reweight
+  T	=  clusinfo          ! include clustering tag in output
+  3.0	=  lhe_version        ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+  T	=  auto_ptj_mjj   ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15	=  bwcutoff       ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+  F	=  cut_decays     ! Cut decay products 
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0	=  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  10	=  ptj        ! minimum pt for the jets 
+  0	=  ptb        ! minimum pt for the b 
```

Comment:
please synchronize j and b related cuts. Otherwise it can create some weird distortions in phasespace

Reply:
Changed to. 10	=  ptb        ! minimum pt for the b